lib/agent/ocsp_response_cache.js

```diff
@@ -136,6 +140,30 @@ function OcspResponseCache()
     downloadStatus = status.FINISHED;
   };
 
+  /**
+   * Is local OCSP Cache expired?
+   * @returns {boolean}
+   */
+  this.IsCacheExpired = function ()
+  {
+    if (!cacheInitialized)
+    {
+      return false;
+    }
+
+    const currentTime = Date.now() / 1000;
+
+    if ((currentTime - cacheUpdateTime) > maxAge)
```

Comment:
Are currentTime, cacheUpdateTime and maxAge in same time unit? I think first two are in milli seconds and maxAge is in seconds. Please double check and put unit in comments for variables.

Reply:
Will add comments for unit.
Date.now() returns milliseconds so `currentTime = Date.now() / 1000` will be in seconds.
cacheUpdateTime is initialized with `Date.now() / 1000` as well so unit is seconds. It could be updated from the timestamp in local cache file, which is made here using currentTime.
https://github.com/snowflakedb/snowflake-connector-nodejs/blob/master/lib/agent/ocsp_response_cache.js#L155